.gitignore

```diff
@@ -3,3 +3,4 @@
 test*.py
 Krakatau/plugins/*
 tests/.cache
+.idea/
```

Comment:
?

Reply:
Intellij IDEA settings. Some files can be committed but this just ignores the folder.